bjoern/server.c

```diff
@@ -278,23 +315,35 @@ ev_io_on_write(struct ev_loop* mainloop, ev_io* watcher, const int events)
     write_state = on_write_chunk(mainloop, request);
   }
 
+  write_state = request->state.expect_continue
```

Comment:
Could you add a little bit of documentation on HTTP 100 handling to the code (a few sentences should be enough). Maybe to the `ev_io_on_write` docstring? Or where you think it fits.